ros2_numpy/point_cloud2.py

```diff
@@ -159,7 +152,7 @@ def array_to_pointcloud2(cloud_arr, stamp=None, frame_id=None):
     cloud_msg.height = cloud_arr.shape[0]
     cloud_msg.width = cloud_arr.shape[1]
     cloud_msg.fields = dtype_to_fields(cloud_arr.dtype)
-    cloud_msg.is_bigendian = False # assumption
+    cloud_msg.is_bigendian = sys.byteorder != 'little',
```

Comment:
The trailing `,` here results in setting `cloud_msg.is_bigendian` to a `tuple` and not a `bool` as you intend.

Reply:
Oh this was a leftover from setting it in the constructor.